docs/ask-cfpb.md

```diff
@@ -0,0 +1,64 @@
+# Ask CFPB API
+
+This API provides search access to the English and Spanish content behind [Ask CFPB](https://www.consumerfinance.gov/ask-cfpb/) and [Obtener respuestas](https://www.consumerfinance.gov/es/obtener-respuestas/).
+
+The financial topics covered include:
+
+- Auto loans
+- Bank accounts and services
+- Credit cards
+- Credit reports and scores
+- Debt collection
+- Families and money
+- Money transfers
+- Mortgages
+- Payday loans
+- Prepaid cards
+- Student loans
+
+## `Usage`
+The API is a read-only resource that delivers search results in [json](https://www.json.org/) format.
+
+Requests follow this pattern:
+
+- `https://www.consumerfinance.gov/ask-cfpb/search/json/?q=[SEARCH TERMS]`
+
+The json response will includes a list of results, each with a question, an answer, and a URL for the related CFPB page. The payload for the search term "tuition" would look like this, but with more result entries:
+
+```json
+{
+  query: "tuition",
+  result_query: "tuition",
+  results: [
```

Comment:
After our typo suggestion work, it will also include `suggestion` at the top level of the json object, and if the search term has no results, a suggested term will be there instead. If there is no suggestion, it will be `null`.

https://www.consumerfinance.gov/ask-cfpb/search/json/?q=tuition

Reply:
Thanks for the suggestion catch! preview is updated:
https://higs4281.github.io/cfgov-refresh/ask-cfpb/